tests/conftest.py

```diff
@@ -42,9 +42,11 @@ def coordinator_service():
     """
 
     server = grpc.server(futures.ThreadPoolExecutor(max_workers=1))
-    coordinator = Coordinator(minimum_participants_in_round=10, fraction_of_participants=1.0)
     store = FakeS3Store()
```

Comment:
should we now use `DummyStore` for tests rather than `FakeS3Store`? I understand you need the latter for `test_full_training_round` but for the rest, the former is probably better.

Reply:
You're right, I changed that.